### PR TITLE
Add/e2e doc

### PR DESCRIPTION
**Pull request by geekgonecrazy**

### CODE REVIEW

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]
Can you add this file to the table of contents?

**Comment by MartinSchoeler** [comment]
Also it might be nice to have a link to this page in here https://github.com/RocketChat/docs/blob/master/user-guides/README.md

**Comment by MartinSchoeler** [comment]
You can also create a new type of channel in [this](https://github.com/RocketChat/docs/tree/master/user-guides/channels) article and link it to the e2e article

**Review by MartinSchoeler — APPROVED** [review]

**Comment by MartinSchoeler** [comment]
Looks good to me!

**Comment by geekgonecrazy** [comment]
Thanks!  🤗

File: user-guides/end-to-end-encryption/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Once you have this password saved somewhere it is safe to dismiss the alert.
+
+## Changing Encryption Password
+
+If you would like to change your encryption password you can do so by going to: My Account->Encryption
+
+![ ](e2e-changepassword.png)
+
+Once you make this change you will need to update the password on any other devices being used.
```
Comment (rodrigok) [inline user-guides/end-to-end-encryption/README.md:25]: This part is not true, the user will need the new password only when importing the credentials again, usually in a new client

Comment (geekgonecrazy) [reply user-guides/end-to-end-encryption/README.md:25]: ah.. good to know! i'll update

**Pull request merged**